tests/ParserSpecTest.php

```diff
@@ -32,6 +32,7 @@ public function specWithoutValueAndDefaultValueProvider()
             [new JsonParser($this->createExceptionFactory(), 'config', null), ['value' => true]],
             [new YesNoBooleanParser($this->createExceptionFactory(), 'isAwesome', null), true],
             [new BooleanParser($this->createExceptionFactory(), 'isNice', null), true]
+            // TODO: [spec, default-value] with comma-separated data types
```

Comment:
Good question, they are tested in [another way here](https://github.com/mpscholten/request-parser/blob/e96f31937b6011b15a653fd77fb5203486d9a26e/tests/CommaSeparatedTypeParserTest.php), but I guess we can cover even more by adding spec tests